identifiers/country-us/state-nc-local_gov.csv

```diff
@@ -260,3 +260,181 @@ ocd-division/country:us/state:nc/place:cary/council_district:a,Cary NC city coun
+ocd-division/country:us/state:nc/county:surry/place:elkin/school_board_district:east,Elkin City Schools Board Of Education East District
+ocd-division/country:us/state:nc/county:surry/place:mount_airy/school_board_district:a,Mount Airy City Schools Board Of Education District A
+ocd-division/country:us/state:nc/county:surry/place:mount_airy/school_board_district:b,Mount Airy City Schools Board Of Education District B
+ocd-division/country:us/state:nc/county:surry/place:mount_airy/school_board_district:c,Mount Airy City Schools Board Of Education District C
+ocd-division/country:us/state:nc/county:surry/place:mount_airy/school_board_district:d,Mount Airy City Schools Board Of Education District D
+ocd-division/country:us/state:nc/county:union/school_board_district:1,Union County School Board District 1
+ocd-division/country:us/state:nc/county:union/school_board_district:2,Union County School Board District 2
+ocd-division/country:us/state:nc/county:union/school_board_district:5,Union County School Board District 5
+ocd-division/country:us/state:nc/county:union/school_board_district:6,Union County School Board District 6
+ocd-division/country:us/state:nc/county:wake/school_board_district:1,Wake County School Board District 1
+ocd-division/country:us/state:nc/county:wake/school_board_district:2,Wake County School Board District 2
+ocd-division/country:us/state:nc/county:wake/school_board_district:3,Wake County School Board District 3
+ocd-division/country:us/state:nc/county:wake/school_board_district:4,Wake County School Board District 4
+ocd-division/country:us/state:nc/county:wake/school_board_district:5,Wake County School Board District 5
+ocd-division/country:us/state:nc/county:wake/school_board_district:6,Wake County School Board District 6
+ocd-division/country:us/state:nc/county:wake/school_board_district:7,Wake County School Board District 7
+ocd-division/country:us/state:nc/county:wake/school_board_district:8,Wake County School Board District 8
+ocd-division/country:us/state:nc/county:wake/school_board_district:9,Wake County School Board District 9
+ocd-division/country:us/state:nc/county:warren/school_board_district:3,Warren County Board Of Education-district 3
+ocd-division/country:us/state:nc/county:warren/school_board_district:5,Warren County Board Of Education-district 5
+ocd-division/country:us/state:nc/county:washington/school_board_district:3,Washington County Board Of Education District 3
+ocd-division/country:us/state:nc/county:wayne/school_board_district:3,Wayne County School Board District 3
+ocd-division/country:us/state:nc/county:wilson/school_board_district:2,Wilson County School Board District 2
+ocd-division/country:us/state:nc/county:wilson/school_board_district:4,Wilson County School Board District 4
+ocd-division/country:us/state:nc/county:wilson/school_board_district:6,Wilson County School Board District 6
+ocd-division/country:us/state:nc/county:cabarrus/place:kannapolis,Kannapolis city
```

Comment:
This is the same capitalization as in the auto-generated census places file. Once I discovered that, it seems to me that we shall not add this as a new OCD ID (under cabarrus county), so I removed it.